reference/build_helpers/cmake.rst

```diff
@@ -292,3 +292,41 @@ Environment variables
 
 There are some environment variables that will also affect the ``CMake()`` helper class. Check them in the
 :ref:`CMAKE RELATED VARIABLES<cmake_related_variables>` section.
+
+Example
+-------
+The following example of ``conanfile.py`` shows you how to manage a project with conan and CMake.
+
+.. code-block:: python
+
+    from conans import ConanFile, CMake
+
+    class SomePackage(ConanFile):
+        name = "SomePackage"
+        version = "1.0.0"
+        settings = "os", "compiler", "build_type", "arch"
+        generators = "cmake"
+
+    def configure_cmake(self):
+        cmake = CMake(self)
```

Comment:
IMO, `set_cmake_flags=True` should be mentioned more prominently in the docs, and this is a good opportunity. It's the kind of option that I thought should be default behavior as newbie.

Reply:
It is well documented here https://docs.conan.io/en/latest/reference/build_helpers/cmake.html#constructor and has not raised any particular issue so far on its default value

Reply:
Sorry, I was off-base on the current documentation context.